### PR TITLE
fjsoft.MyPhoneExplorer version 1.8.15

### PR DESCRIPTION
**Pull request by donid**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16952)

### CODE REVIEW

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
+  InstallerUrl: https://www.fjsoft.at/download.php?id=1
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.installer.yaml:8]:
```suggestion
  InstallerUrl: https://www.fjsoft.atfiles/MyPhoneExplorer_Setup_1.8.15.exe
```
Please use static URLs if possible.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer Desktop
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:7]:
```suggestion
PackageName: MyPhoneExplorer
```
It shows as MyPhoneExplorer in Control Panel.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer Desktop
+License: © fjsoft.at
+ShortDescription: Verbinden Sie Ihr Handy über WLAN, USB-Kabel oder Bluetooth mit Ihrem PC und Sie werden überrascht sein, wie einfach und effizient Sie Ihr Smartphone mit diesem kompakten Programm verwalten können!
+ManifestType: defaultLocale
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:10]:
```suggestion
ManifestType: locale
```
Please use English as the default locale.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: de
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:5]:
```suggestion
PackageLocale: de-DE
```
Invalid Locale code.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.yaml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+DefaultLocale: de
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.yaml:5]:
```suggestion
DefaultLocale: en-US
```
Invalid locale Code.

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Please create a English locale `fjsoft.MyPhoneExplorer.locale.en-US.yaml`
And rename `fjsoft.MyPhoneExplorer.locale.de.yaml` to `fjsoft.MyPhoneExplorer.locale.de-DE.yaml`

You should use GitHub Desktop for this.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml (hunk 4 above)
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:5]:
```suggestion
PackageLocale: de-DE
```
Invalid locale Code

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml (hunk 4 above)
Comment (donid) [reply manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:5]: Well, the developer is from Austria, so I am not sure if de-DE or de-AT would be appropriate

Comment (OfficialEsco) [reply manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de.yaml:5]: It does not matter where the Developer is from, the only thing that matters is what language the Metadata is in. Aka your ShortDescription/Description. And Homepage, License, LicenseUrl, PublisherUrl, PublisherSupportUrl, Copyright and CopyrightUrl if that gets added.

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de-DE.yaml [hunk 6]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: "de-DE"
+PackageURL: "https://www.fjsoft.at/de/"
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.de-DE.yaml:6]:
```suggestion
PackageUrl: "https://www.fjsoft.at/de/"
```

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate
+
+PackageIdentifier: fjsoft.MyPhoneExplorer
+PackageVersion: 1.8.15
+PackageLocale: "en-US"
+Publisher: F.J. Wechselberger
+PackageName: MyPhoneExplorer
+PackageURL: "https://www.fjsoft.at/en/"
```
Comment (OfficialEsco) [inline manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml:8]:
```suggestion
PackageUrl: "https://www.fjsoft.at/en/"
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
`Manifest Error: All field names should be PascalCased. Field: PackageURL Manifest Error: All field names should be PascalCased. Field: PackageURL`

File: manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml (hunk 7 above)
Comment (donid) [reply manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml]: I copied the field from the docs... Yaml should already be fixed and I opened a Issue: https://github.com/MicrosoftDocs/windows-uwp/issues/3167

Comment (OfficialEsco) [reply manifests/f/fjsoft/MyPhoneExplorer/1.8.15/fjsoft.MyPhoneExplorer.locale.en-US.yaml]: Daaamn you're right, i knew i've seen PackageURL before and i've typed it myself at times, good catch 👍

**Review by OfficialEsco — APPROVED** [review]

**Pull request merged**